o1vm/src/mips/column.rs

```diff
@@ -14,14 +14,22 @@ use strum::EnumCount;
 
 use super::{ITypeInstruction, JTypeInstruction, RTypeInstruction};
 
-pub(crate) const MIPS_HASH_COUNTER_OFFSET: usize = 80;
-pub(crate) const MIPS_IS_SYSCALL_OFFSET: usize = 81;
-pub(crate) const MIPS_READING_PREIMAGE_OFFSET: usize = 82;
-pub(crate) const MIPS_BYTES_READ_OFFSET: usize = 83;
-pub(crate) const MIPS_PREIMAGE_LEFT_OFFSET: usize = 84;
-pub(crate) const MIPS_PREIMAGE_BYTES_OFFSET: usize = 85;
-pub(crate) const MIPS_HAS_N_BYTES_OFFSET: usize = 89;
-pub(crate) const MIPS_CHUNK_BYTES_LENGTH: usize = 4;
+/// The number of hashes performed so far in the block
+pub(crate) const MIPS_HASH_COUNTER_OFF: usize = 80;
+/// The number of bytes of the preimage that have been read so far in this hash
+pub(crate) const MIPS_BYTE_COUNTER_OFF: usize = 81;
+/// A flag indicating whether the preimage has been read fully or not
+pub(crate) const MIPS_END_OF_PREIMAGE_OFF: usize = 82;
+/// The number of preimage bytes processed in this step
+pub(crate) const MIPS_NUM_BYTES_READ_OFF: usize = 83;
+/// The at most 4-byte chunk of the preimage that has been read in this step
+pub(crate) const MIPS_PREIMAGE_CHUNK_OFF: usize = 84;
+/// The at most 4-bytes of the preimage that are currently being processed
+pub(crate) const MIPS_PREIMAGE_BYTES_OFF: usize = 85;
+/// Flags indicating whether at least N bytes have been processed in this step
+pub(crate) const MIPS_HAS_N_BYTES_OFF: usize = 89;
+/// The maximum size of a chunk (4 bytes)
```

Comment:
Nit (sorry, very "stupid"): only maybe adding a small notes saying that the values above are offsets for the registers, and this one is simply a constant.

Reply:
Also, i do not understand why this variable is in `columns.rs`. Why not in libs.rs or somewhere else?